### DISCUSSION TITLE
Node.js 18

### DISCUSSION THREAD

**Discussion by JackTelford** [DustinBrett/daedalOS#448] [General]
Suggesting to add in the readMe you must use Node.js 18 for the project since node.js is at 20.

**Comment by DustinBrett** [comment]
I run on Node v21 so v18 is not a requirement. What makes you think it is?

If you are running into an error about `digital envelope routines::unsupported` that is covered in the readme

https://github.com/DustinBrett/daedalOS?tab=readme-ov-file#notes
